src/features/controls.ts

```diff
@@ -239,21 +249,15 @@ export const ControlsMixin = (ModelViewerElement:
         }
 
         [$updateFieldOfView]() {
-          let fov = deserializeAngleToDeg(this.fieldOfView);
+          let fov = deserializeAngle(this.fieldOfView);
```

Comment:
Might be best to include the second argument here to make the units more clear.

Reply:
Good call, will fix.